package.json

```diff
@@ -8,19 +8,41 @@
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
     "test": "react-scripts test --watchAll=false",
+    "test:coverage": "react-scripts test --coverage --watchAll=false",
+    "test:update": "react-scripts test --updateSnapshot --watchAll=false",
     "coveralls": "react-scripts test --coverage --watchAll=false && cat ./coverage/lcov.info | coveralls",
-    "test:coverage": "react-scripts test --coverage --watchAll=false"
+    "storybook": "start-storybook",
+    "build-storybook": "build-storybook -c .storybook -o docs"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "**/*.{js,jsx}",
+      "!**/node_modules/**",
+      "!**/*.stories.{js,jsx}",
+      "!src/components/**/index.{js,jsx}",
+      "!src/pages/**/index.{js,jsx}",
+      "!src/config/**"
+    ]
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run lint"
     }
   },
   "dependencies": {
-    "react": "^16.9.0",
-    "react-dom": "^16.9.0",
+    "@material-ui/core": "^4.5.1",
+    "@material-ui/icons": "^4.5.1",
+    "@material-ui/styles": "^4.5.0",
+    "@storybook/addon-actions": "^5.2.4",
+    "@storybook/addon-notes": "^5.2.4",
+    "@storybook/react": "^5.2.4",
```

Comment:
Can we move storybook to devDependencies?

Reply:
agree